component.js

```diff
@@ -47,6 +47,7 @@ module.exports = factory();
  *   shouldComponentUpdate: function(nextProps, nextState), // check update
  *   cursorField: '__singleCursor', // cursor property name to "unwrap" before passing in to render
  *   isNode: function(propValue), // determines if propValue is a valid React node
+ *   classDecorator: function(Component), // Allows for decorating created class
```

Comment:
Will we be decorating other stuff than the class, or is `decorator` enough? Maybe.. don't know what that would be at the moment, though